source/tutorials/integration-testing-using-virtual-machines.md

```diff
@@ -1,218 +0,0 @@
-(integration-testing-vms)=
```

Comment:
nixos-configuration-testing.md in this PR has the same content as integration-testing-using-virtual-machines.md in #305 . I didn't fixed the discrepancy because I didn't expect a review on a draft.